src/Config_Command.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use EE\Model\Site;
+use Mustangostang\Spyc;
+use \Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Manges global EE configuration.
+ *
+ * ## EXAMPLES
+ *
+ *     # Save value in config
+ *     $ ee config set le-mail='abc@example.com' admin-email=abcd@example1.com
+ *
+ *     # Get value from config
+ *     $ ee config get le-mail
+ *
+ * @package ee-cli
+ */
+class Config_Command extends EE_Command {
+
+	/**
+	 * @var Filesystem $fs Symfony Filesystem object.
+	 */
+	private $fs;
+
+	public function __construct() {
+
+		$this->fs = new Filesystem();
+	}
+
+	/**
+	 * Get a config value
+	 *
+	 * ## OPTIONS
+	 *
+	 * <config-key>
+	 * : Name of config value to get
+	 */
+	public function get( $args, $assoc_args ) {
+		$config_file_path = getenv( 'EE_CONFIG_PATH' ) ? getenv( 'EE_CONFIG_PATH' ) : EE_CONF_ROOT . '/config.yml';
+		$config = Spyc::YAMLLoad( $config_file_path );
+
+		if ( ! isset( $config[$args[0]] ) ) {
+			EE::error("No config value with key '$args[0]' set");
+		}
+
+		EE::log( $config[$args[0]] );
+	}
+
+	/**
+	 * Get a config value
+	 *
+	 * ## OPTIONS
+	 *
+	 * <config-key-value>...
```

Comment:
why you're changing the structure of command? the command should be like
>  ee config set le-mail abc@example.com

ref: https://developer.wordpress.org/cli/commands/config/set/

if we want `le-mail='abc@example.com'` this then we should use `assoc_args` by using ` [--<field>=<value>]`

Reply:
because of this, you'll have to do manipulation of $args by using `explode` which I feel patch work in respect of our command structure.

Reply:
I had discussed this with @mbtamuli and @mrrobot47 and we felt this was the best syntax to use as --<field>=<value> felt unnatural/unintuitive to use.

Reply:
we don't need to use `--<field>=<value>` I'm said that we should use a proper command structure which we are following do not add other structure which need some manipulation and  patch work.

> ee config set le-mail abc@example.com

just follow this.

Reply:
Will this allow for multiple options to be set in one go?

`ee config set le-mail abc@example.com sites_path /mnt/sites`

Reply:
But this would not allow us to specify multiple values. With current implementation, you can do following 

`ee config set key1=val1 key2=val2 key3=val3`. We wouldn't be able to do this with following syntax:

> ee config set le-mail abc@example.com

Reply:
yes, it will work.

Reply:
`ee config set key1 val1 key2 val2 key3 val3` will work as well as the you'll get all these in the `$args` array. You won't have to use explode.

Reply:
Based on our discussion, let's go for only being able to set one option at a time.

```ee config set le-mail abc@example.com```

Reply:
@rahulsprajapati Done